src/client/src/models/Scene.js

```diff
@@ -17,6 +17,12 @@ const Scene = {
         captions: []
       }]))
     }
+
+    if(!localStorage.getItem('input-format')) {
+      localStorage.setItem('input-format', JSON.stringify({
+        inputFormat: 'CAE-608'
```

Comment:
This seems weird to me to give it the key "inputFormat" when the object in the localStorage is given the key "input-format". That means that when logging out localstorage you will see this:

```
...
input-format: "{"inputFormat":"CAE-608"}"
...
```

We can probably just have the one property "input-format" and have that map to the value "CEA-608"